mitosheet/mitosheet/streamlit/v1/spreadsheet.py

```diff
@@ -122,6 +122,13 @@ def __init__(self, code: str, code_options: Optional[CodeOptions], fully_paramet
     def param_metadata(self) -> List[ParamMetadata]:
         return self.__param_metadata
     
+    def get_param_metadata(self, type: Optional[ParamType]=None):
```

Comment:
we should call it `param_type`, so it doesn't overlap with the builtin python `type` function.